lib/reevoo-ping.js

```diff
@@ -2,13 +2,22 @@
 //
 // Main event recording class.
 
+import { COLLECTOR_URI } from './config';
+import snowplow from './snowplow';
+import defaultSnowplowConfig from './snowplow/default-config';
 import validate from './validate';
 import Experiences from './events/experiences';
 import Page from './events/page';
 import Badge from './events/badge';
 
 class Client {
-  constructor(opts) {
+  constructor(appId, opts = {}, trackerConfig = {}) {
```

Comment:
Yes it will, but it was wrong before, the appId should be required. There are only two projects using reevoo-ping.js so far: mark and aslan.